pkg/pillar/cmd/verifier/lib/logger.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package verifier
+
+// Logger is an interface for logging, combines the best of logrus.Logger and base.LogObject
```

Comment:
If this is useful should we later promote it (e.g., to base.Logger)? No need to touch this PR though.

Reply:
Yeah, that would be a good thing to do. Once this one is in, I will open a separate PR just for this.

I have used this pattern in the past. I never really understood why `log.Logger` (or similar) is not an interface. Just work with what we have, I guess.